EPS/eps_subsystem.py

```diff
@@ -66,14 +75,79 @@ def __init__(self):
         }
         self.updatable_parameters = ['WatchdogResetTime']
 
+        self.subsystems = {
+            'ADCS': False,
+            'Deployables': False,
+            'DFGM': False,
+            'GPS': False,
+            'IRIS': False,
+            'UHF': False
```

Comment:
This is great. Could also add in additional fields for 'GPIO' pins representing burn wires for antennas and UHF specifically.